modules/blake3/1.3.3.bcr.1/patches/add_build_file.patch

```diff
@@ -0,0 +1,65 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,62 @@
++load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
++
++cc_library(
++    name = "blake3",
++    srcs = [
++        "c/blake3.c",
++        "c/blake3_dispatch.c",
++        "c/blake3_portable.c",
++    ] + select({
++        "@bazel_tools//src/conditions:linux_x86_64": [
++            "c/blake3_avx2_x86-64_unix.S",
++            # Disable to appease bazel-ci which uses ubuntu-18 (EOL) and GCC 7
```

Comment:
is this accurate? your presubmit.yml lists ubuntu2004.

Reply:
It's accurate for bazel, which is where I grabbed this BUILD file from.

(If you feel this comment is too problematic I can remove it)

Reply:
not a big deal at all, just wanted to point it out in case you missed something.